test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua

```diff
@@ -0,0 +1,86 @@
+ -----  Name of requirement that is covered-----
+ ----- Certificate have EMPTY in pt_preloaded JSON of module_config section
+ ----- Verification criteria------
+ -----  SDL must continue working as assigned.
+
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]  
+local mobile_session = require('mobile_session')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+NewTestSuiteNumber = 0
+require('user_modules/AppTypes')
+
+--[[ Local preparing ]]
```

Comment:
@Aleksandriys , where block --[[ Local preparing ]] is from?? It should be renamed to --[[ Local functions ]] or --[[ Preconditions ]] - depending on its purpose. We should use block names from our template.